src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -272,12 +272,17 @@ namespace winrt::TerminalApp::implementation
                                 (_tabs.Size() > 1) ||
                                 _settings.GlobalSettings().AlwaysShowTabs());
 
-        // collapse/show the tabs themselves
-        _tabView.Visibility(isVisible ? Visibility::Visible : Visibility::Collapsed);
-
-        // collapse/show the row that the tabs are in.
-        // NaN is the special value XAML uses for "Auto" sizing.
-        _tabRow.Height(isVisible ? NAN : 0);
+        if (_tabView)
+        {
+            // collapse/show the tabs themselves
+            _tabView.Visibility(isVisible ? Visibility::Visible : Visibility::Collapsed);
+        }
+        if (_tabRow)
+        {
+            // collapse/show the row that the tabs are in.
+            // NaN is the special value XAML uses for "Auto" sizing.
+            _tabRow.Height(isVisible ? NAN : 0);
+        }

```

Comment:
This seems a bit unrelated. Did QM initialize without a `_tabView` and `_tabRow` or what happened?

Reply:
> Did QM initialize without a `_tabView` and `_tabRow`

SURE DID